packages/@aws-cdk/tmp-toolkit-helpers/src/api/io/private/messages.ts

```diff
@@ -297,7 +297,7 @@ export const IO = {
     interface: 'ErrorPayload',
   }),
 
-  // 9: Bootstrap
+  // 9: Bootstrap (9xxx)
```

Comment:
i assume by the way you are "enforcing" this that these are not meant to be strongly enforced at all, and will rely on our attention to detail to ensure we follow.

i feel like we are getting to the point where these rules may be hard to get right when we eventually introduce new message codes.